src/site/content/en/fast/incorporate-performance-budgets-into-your-build-tools/index.md

```diff
@@ -13,16 +13,45 @@ codelabs:
 
 Once you’ve defined a performance budget, it’s time to set up the build process
 to keep track of it. There are a number of tools that let you define thresholds
-for chosen performance metrics and will warn you if you go over budget. Find out
+for chosen performance metrics and warn you if you go over budget. Find out
 how to choose one that best fits your needs and current setup. 🕵️‍♀️
 
-## Webpack Performance Hints
+
+## Lighthouse performance budgets
+
+[Lighthouse](https://developers.google.com/web/tools/lighthouse/) is an auditing tool that tests sites in a few key areas — performance, accessibility, best practices and how well your site performs as a progressive web application.
+
+The [command line version](https://developers.google.com/web/tools/lighthouse/#cli) of Lighthouse (v5+) supports setting [performance budgets](https://developers.google.com/web/tools/lighthouse/audits/budgets) based on:
+
+- resource size
+- resource count
+
+You can set budgets for any of the following resource types: 
+- `document` 
+- `font`
+- `image`
+- `media`
+- `other`
+- `script`
+- `stylesheet`
+- `third-party`
+- `total`
+
+Budgets are set in a JSON file and after defining them the new Over Budget column tells you whether you're exceeding any limits.
+
+<figure class="w-figure">
+  <img src="./lightwallet.png" alt="Lighthouse Bot">
```

Comment:
Broken image, `alt` probably also needs to be updated.

Reply:
This is also not supposed to be broken, [here's the image](https://github.com/GoogleChrome/web.dev/blob/perf-budgets/src/site/content/en/fast/incorporate-performance-budgets-into-your-build-tools/lightwallet.png).

Will updated alt when I figure out where are my images 👍